webdriver/tests/bidi/emulation/set_user_agent_override/user_contexts.py

```diff
@@ -0,0 +1,157 @@
+import pytest
+
+pytestmark = pytest.mark.asyncio
+
+SOME_USER_AGENT = "SOME_USER_AGENT"
+
+
+async def test_contexts(bidi_session, new_tab, top_context, default_user_agent,
```

Comment:
I think we don't need this test since we have already the same one in `contexts.py`?